src/Brick/Widgets/List.hs

```diff
@@ -215,7 +230,7 @@ drawListElements foc l drawElem =
 
             drawnElements = flip V.imap es $ \i e ->
                 let isSelected = Just (i + start) == l^.listSelectedL
-                    elemWidget = drawElem isSelected e
+                    elemWidget = drawElem (i + start) isSelected e
```

Comment:
Also please pull `i + start` out as a separate `let` binding such as `idx` since it is now used in multiple places.